docs/content/preview/migrate/performance.md

```diff
@@ -24,39 +24,41 @@ There are several factors that slow down data-ingestion performance in any datab
 
 yb-voyager maximizes performance when migrating data into a newly created empty database in several ways:
 
-- Creates secondary indexes after the data import is complete. During the [import data](../migrate-steps/#import-data) phase, yb-voyager creates the secondary indexes after it completes data loading on all the tables. Then, it uses [Index Backfill](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/online-index-backfill.md) to update indexes after data is loaded. This is much faster than online index maintenance.
+- Creates secondary indexes after the data import is complete. During the [post-import data](../migrate-steps/#import-indexes-and-triggers) phase, yb-voyager creates the secondary indexes (except the unique indexes) after it completes data loading on all the tables. Then, it uses [Index Backfill](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/online-index-backfill.md) to update indexes after data is loaded. This is much faster than online index maintenance. (Unique indexes are created in the [import-schema](../migrate-steps/#import-schema) phase, to avoid any foreign key dependencies on the index.)
 
-- Disables constraint checks. It's safe to disable all the constraint checks provided the data is from a reliable source. For maximum throughput, it is also preferable to not follow any order when populating tables. During the data import phase, yb-voyager disables all constraints, except for primary key violation errors.
+- Disables constraint checks except check constraints on the column. It's safe to disable all the constraint checks provided the data is from a reliable source. For maximum throughput, it is also preferable to not follow any order when populating tables. During the data import phase, yb-voyager disables all constraints, except for primary key violation errors.
 
   {{< note title="Note" >}}
 
-yb-voyager only disables all the constraints in internal sessions to migrate data.
+yb-voyager only disables constraint checks (except check constraints on columns) in internal sessions to migrate data.
```

Comment:
Please take this _note_ section below the _Disables triggers_ point and modify the note content like this.

> yb-voyager only disables the constraint checks and triggers in the internal sessions it uses to migrate data.

Reply:
Sorry missed seeing this one and the top one. Thanks for pointing it out.